index.html

```diff
@@ -7,16 +7,37 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Awesome Books</title>
   <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
     integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
 </head>
 
 <body>
-  <h1 class="catch">Awesome Books</h1>
+<div>
+<div class="topnav">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">Programming</a>
+  <a href="#contact">Space Sciences</a>
+  <a href="#about">About</a>
+</div>
+</div>
   <div class="container">
-    <h1 class="display"></h1>
-    <div class="books">
+    <div class="display">
+      <h1>All Awesome Books</h1>
     </div>
+    <table class="table table-striped books">
+      <thead>
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Title</th>
+          <th scope="col">Author</th>
+          <th scope="col"></th>
+        </tr>
+      </thead>
+      <tbody>
+      </tbody>
+    </table>
     <br>
     <form id="book-form">
       <label for="title" class="form-label">Title</label>

```

Comment:
- [ ] The Add button must be placed on the right side, not in the left side

Your layout:

![image](https://user-images.githubusercontent.com/60631456/124786440-534c2480-df0d-11eb-9774-1ee9d928748f.png)

Original layout:

<img width="437" alt="awesome_books_core_elements" src="https://user-images.githubusercontent.com/60631456/124786709-8db5c180-df0d-11eb-9aa4-915986229dec.png">